client/components/thank-you/index.tsx

```diff
@@ -177,8 +177,8 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 		background-color: ${ headerBackgroundColor };
 		min-height: 352px;
 		img {
-			width: auto;
-			height: 200px;
+			width: ${ thankYouImage.width ? null : 'auto' };
+			height: ${ thankYouImage.height ? null : '200px' };
```

Comment:
This allows us to pass in custom width and height values that get set on the `<img>` element as necessary.